Vishnu_45/html/Assignment3/ass2_home.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
```

Comment:
need proper code indentation
remove all extra lines
add new line
change all with at the end of css property ends